bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/handler/SamsungTvHandler.java

```diff
@@ -273,7 +277,8 @@ private synchronized void checkCreateManualConnection() {
                     RemoteControllerService.SERVICE_NAME);
             if (service == null) {
                 SamsungTvConfiguration configuration = getConfigAs(SamsungTvConfiguration.class);
```

Comment:
Because configuration is not anymore updated and getConfigAs is rather expensive call (it' called in many places), I think it should be read only ones on initialisation (like it have been before).

Reply:
Ok, this is now changed.